osu.Game/IO/OsuCachedOnlineStore.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.IO;
+using osu.Framework.IO.Stores;
+
+namespace osu.Game.IO
+{
+    public abstract class OsuCachedOnlineStore : CachedOnlineStore
+    {
+        protected override string CachePath => Path.Combine(Path.GetTempPath(), "osu");
```

Comment:
This is incorrect and should be osu! installation directory("C:\Users\TocoT\AppData\Roaming\osu\cache" on my machine), but I don't know how to get the correct path.